nlp/twitter.py

```diff
@@ -2,106 +2,34 @@
 import re
 import pickle
 import signal
-from twython import Twython
-from twython import TwythonStreamer
+from twython import Twython, TwythonStreamer, exceptions
+from argparse import ArgumentParser
+from pathlib import Path
 
-emoji_dict = dict()
+# used in order to open twitter stream automatically
+import urllib
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.chrome.options import Options
 
-# Twitter Setup
-TWITTER_KEY = os.environ.get('TWITTER_KEY')
-TWITTER_SECRET = os.environ.get('TWITTER_SECRET')
+import constants
 
-'''
-Define emoji unicode here
-Examples:
-1F602 = happy, 1F62D = sad, 1F621= angry, 2764 = love, 1F61C = playful, 1F631 = confused
-'''
-EMOJIS = [u'\U0001F621']
+EMOJIS = ['😡']
```

Comment:
emojis work, btw! i looked into this in the twython and twitter api documentation and it supports searching for emojis. this should make it easier for you to search for different emojis now. 🙂